components/pipeline-service/production/base/main-pipeline-service-configuration.yaml

```
@@ -1861,6 +1861,8 @@ spec:
     resources:
       - pipelinerun
     schedule: '*/10 * * * *'
+  results:
```

```suggestion
  result:
```
see here: https://github.com/redhat-appstudio/infra-deployments/blob/main/components/pipeline-service/development/main-pipeline-service-configuration.yaml#L2053